app/views/admin/locations/_form.html.erb

```diff
@@ -13,15 +13,17 @@
 
   <div>
     <%= form.label :latitude, style: "display: block" %>
-    <%= form.text_field :latitude %>
+    <%= form.text_field :latitude, class: "form-control", id: "InputLatitude", placeholder: "Enter latitude" %>
   </div>
 
   <div>
     <%= form.label :longitude, style: "display: block" %>
-    <%= form.text_field :longitude %>
+    <%= form.text_field :longitude, class: "form-control", id: "InputLongitude", placeholder: "Enter longitude" %>
   </div>
 
   <div>
-    <%= form.submit %>
+    <button type="submit" class="btn btn-dark">
```

Comment:
@lime666 use Rails form helpers instead of HTML, like `<%= form.submit 'Submit', class: 'btn btn-dark' %>`